Makefile

```diff
@@ -5,7 +5,7 @@ CFLAGS-DEBUG=-O0 -Wall
 BIN=./bin
 
 SUDO=sudo
-MKLIB=$(SUDO) bash ./libs/install.sh ./libs
+MKLIB=bash ./libs/install.sh ./libs
```

Comment:
Remember my rant on #29 is about `make build` trigger put files on system, which I expect this only on `make install`. I look the path of libs has been changed and we avoid the usage of sudo for now. But we still writing files on build step. Is possible separate the `build` process to get the binaries and the `install` for copying them? 

The word `mixed` on issue is just about that.